src/RestEaseUnitTests/ImplementationBuilderTests/PathParamTests.cs

```diff
@@ -329,6 +358,48 @@ public void SerializeToStringUsesGivenFormatProvider()
             formatProvider.Verify(x => x.GetFormat(typeof(NumberFormatInfo)));
         }
 
+        [Fact]
+        public void RecordsSerializedSerializationMethod()
+        {
+            var requestInfo = Request<ISerializedPathParam>(x => x.FooAsync("fizzbuzz"));
+
+            Assert.Single(requestInfo.PathParams);
+            Assert.Equal(PathSerializationMethod.Serialized, requestInfo.PathParams.First().SerializationMethod);
```

Comment:
Nit: `[0]`

Reply:
That doesn't compile (`IEnumerable` does not implement an indexer).

Reply:
Right - I was looking at QueryParamTests, where we call ToList. I guess it doesn't matter though.